dbx/encrypt-fields.rst

```diff
@@ -11,58 +11,79 @@ In-Use Encryption
+
+- :ref:`Queryable Encryption <subsection-qe>`
+- :ref:`Client-side Field Level Encryption <subsection-csfle>`
+
+.. _subsection-qe:
 
 Queryable Encryption
 ~~~~~~~~~~~~~~~~~~~~
 
 Queryable Encryption is the next-generation in-use encryption feature,
-introduced in MongoDB Server version 6.0 and available as a public
-preview. Queryable Encryption supports searching encrypted fields for
-equality and encrypts each value uniquely.
+first introduced as a preview feature in MongoDB Server version 6.0 and
+as a generally available (GA) feature in MongoDB 7.0. Queryable
+Encryption supports searching encrypted fields for equality and encrypts
+each value uniquely.
+
+.. important:: Preview Feature Incompatible with MongoDB 7.0
+   
+   The implementation of Queryable Encryption in MongoDB 6.0 is incompatible with the GA version introduced in MongoDB 7.0. The Queryable Encryption preview feature is no longer supported.
 
-The MongoDB manual contains detailed information on the following Queryable Encryption topics:
+To learn more about Queryable Encryption, see the :manual:`Queryable
+Encryption </core/queryable-encryption/>` documentation in the Server manual.
 
-- To get started, see the :ref:`Queryable Encryption Quick Start <qe-quick-start>`.
-- To learn how to use Queryable Encryption, see the :ref:`Queryable Encryption Fundamentals <qe-fundamentals>`.
-- To learn how to integrate your implementation with a Key Management System, see the :ref:`Queryable Encryption Tutorials <qe-tutorials>`.
-- To learn Queryable Encryption concepts, see the :ref:`Queryable Encryption Reference <qe-reference>`.
+.. _subsection-csfle:
 
 Client-side Field Level Encryption
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Client-side Field Level Encryption (CSFLE) was introduced in MongoDB
 Server version 4.2 and supports searching encrypted fields for equality.
-CSFLE differs from Queryable Encryption in that it requires that the encrypted fields
-you want to search must be deterministically encrypted. When you
-deterministically encrypt a value, the same input value produces the
-same output value. While deterministic encryption provides greater 
-support for read operations, encrypted data with low :wikipedia:`cardinality <Cardinality>`
-is susceptible to recovery using :wikipedia:`frequency analysis <Frequency_analysis>`.
-
-The MongoDB manual contains detailed information on the following CSFLE topics:
-
-- To get started, see the :ref:`CSFLE Quick Start <csfle-quick-start>`.
-- To learn how to use CSFLE, see the :ref:`CSFLE Fundamentals <csfle-fundamentals>`.
-- To learn how to integrate your CSFLE implementation with a Key Management System, see the :ref:`CSFLE Tutorials <csfle-tutorials>`.
-- To learn CSFLE concepts, see the :ref:`CSFLE Reference <csfle-reference>`.
+CSFLE differs from Queryable Encryption in that you can select either a
+deterministic or random encryption algorithm to encrypt fields. You only
```

Comment:
I think "You can only" would convey the limitation better than "you only", which is more of a description of a behavior.